Source/WarpX.H

```diff
@@ -322,18 +362,21 @@ public:
     void EvolveE (int lev, amrex::Real dt);
     void EvolveB (         amrex::Real dt);
     void EvolveB (int lev, amrex::Real dt);
-    void EvolveM (         amrex::Real dt);
-    void EvolveM (int lev, amrex::Real dt); // declare WarpX::EvolveM, defined in WarpXPushFieldsEM.cpp
     void EvolveF (         amrex::Real dt, DtType dt_type);
     void EvolveF (int lev, amrex::Real dt, DtType dt_type);
     void EvolveB (int lev, PatchType patch_type, amrex::Real dt);
     void EvolveE (int lev, PatchType patch_type, amrex::Real dt);
-    void EvolveM (int lev, PatchType patch_type, amrex::Real dt);
     void EvolveF (int lev, PatchType patch_type, amrex::Real dt, DtType dt_type);
 
     void MacroscopicEvolveE (         amrex::Real dt);
     void MacroscopicEvolveE (int lev, amrex::Real dt);
     void MacroscopicEvolveE (int lev, PatchType patch_type, amrex::Real dt);
+#ifdef WARPX_MAG_LLG
+    void MacroscopicEvolveM (         amrex::Real dt);
+    void MacroscopicEvolveM (int lev, amrex::Real dt);
+    void MacroscopicEvolveM (int lev, PatchType patch_type, amrex::Real dt);
+#endif
+    void FieldGather ();
```

Comment:
This FieldGather () function is not there in Source/WarpX.H and not part of the PR. this can be deleted. probably a left over from some merge?